### PR TITLE
Feature/449

### PR DESCRIPTION
**Pull request by AlanBCole**
added some lines of code into cu_core.install to allow overridden features to appear in the status page.

### CODE REVIEW

**Comment by kevincrafts** [comment]
When I reviewed the PR I did not see the overriden Sitewide context listed on the report page - just an item labeled as an array.  When I reverted my sitewide feature to default, that listing disappeared.

Screen Shot 2020-03-19 at 11 35 35 AM

**Comment by AlanBCole** [comment]
@kevincrafts Hmm. What I see is:

Screen Shot 2020-03-19 at 12 07 20 PM

I am guessing this is due to some differences in my local environment? The overridden features listed in the image are the same (for me) as found in:
- admin/structure/features
- drush fl. 

Not sure what to do.

**Comment by kevincrafts** [comment]
I also wonder if there are features that we want to exempt from this list, since they are more or less designed to be overridden. No need to have a site report say something is wrong because the sitewide context is overridden when that is probably going to be the case for a lot of sites.

**Comment by AlanBCole** [comment]
That would be fairly easy to do, I think. It is also very easy to show the features in the status page not as warnings but rather just at the information level. I think that is the level of the blue rows.

**Comment by AlanBCole** [comment]
Just pushed little fixes so the code passes the phpcodesniffer.

**Comment by AlanBCole** [comment]
Is there a definite list of features that we know ahead of time will be overridden or is we will have to create dynamically on each page load?

**Comment by cathysnider** [comment]
Locally, my list of overridden feature is blue. That doesn't seem like the right color for a warning. 

Screen Shot 2020-03-20 at 4 23 35 PM

**Comment by AlanBCole** [comment]
### Requested change fixes
- switch severity level to SEVERITY_WARNING (yellow with a little flag)
- put all overridden features on on line
- ignore features known to be ovverridden
  - at least a preliminary fix for this

File: modules/custom/cu_core/cu_core.install [hunk 1]
```diff
@@ -340,8 +340,36 @@ function cu_core_requirements($phase) {
...
+    // Just put the sting name of the feature in.
+    $featureStatesToIgnore = [
+      'cu_sitewide',
+      'cu_sitemap',
+      'cu_feature_callout',
+      'cu_social_links',
+      'cu_webform',
+      'cu_mega_menu',
+      'cu_ldap',
+      'other features to overlook...'
```
Comment (jwfuller) [inline modules/custom/cu_core/cu_core.install]: Remove this line

File: modules/custom/cu_core/cu_core.install [hunk 2]
```diff
@@ -340,8 +340,36 @@ function cu_core_requirements($phase) {
...
+
+    if (count($overriddenFeatures) > 0) {
+      $requirements['overridden_features'] = [
+        'title' => $t('Overridden Features'),
+        'value' => t(implode(', ', $overriddenFeatures)),
+        'severity' => REQUIREMENT_ERROR,
+      ];
+    }
 
-    // Get enabled modules with hosting in the name.
```
Comment (jwfuller) [inline modules/custom/cu_core/cu_core.install]: Restore this comment

File: modules/custom/cu_core/cu_core.install (hunk 1 above)
Comment (AlanBCole) [reply modules/custom/cu_core/cu_core.install]: I am assuming you are referring to only line 354.

**Pull request merged**